include/triton/Dialect/TritonGPU/IR/TritonGPUOps.td

```diff
@@ -207,4 +208,19 @@ def TTG_AllocTensorOp : TTG_Op<"alloc_tensor", [MemoryEffects<[MemAlloc]>,  // A
   let results = (outs TT_Tensor:$result);
 }
 
+def TTG_TensorToMemRefOp : TTG_Op<"tensor_to_memref", [SameOperandsAndResultShape,
+                                                       SameOperandsAndResultElementType,
+                                                       ResultsAreSharedEncoding,
```

Comment:
This trait needs to be removed.

Reply:
The ResultsAreSharedEncoding can be remove, since we are using LDSEncoding. The rest might be used for analysis.